docs/UserGuide.md

```diff
@@ -139,7 +139,8 @@ Section 2.2 will explain the features related to items.
 
 #### 2.2.1. Show items-related commands: `items`
 
-Show commands related to items and brief description of the commands: add item, list items, remove items, clear items
+As a first-time user, you can simply type `items` to view all commands related to items. At the same time, you can see
+the brief description of those commands.
 
```

Comment:
same as above, but i feel this one is more relevant towards just looking at items related features if the user forgets